src/Sentry.Unity/SentryUnity.cs

```diff
@@ -64,6 +65,18 @@ public static void Init(SentryUnityOptions options)
                             new ScreenshotAttachmentContent(options, SentryMonoBehaviour.Instance))));
                 }
 
+                if (options.NativeContextWriter is { } contextWriter)
+                {
+                    try
+                    {
+                        SentrySdk.ConfigureScopeAsync((scope) => Task.Run(() => contextWriter.Write(scope)));
```

Comment:
Since we're not awaiting on the task returned by `ConfigureScopeAsync` it's the same as:
```suggestion
                        SentrySdk.ConfigureScope((scope) => Task.Run(() => contextWriter.Write(scope)));
```